dydx/dydxCartera/dydxCartera/WalletSetup/dydxWalletSetup.swift

```diff
@@ -72,7 +72,7 @@ public class dydxWalletSetup: WalletStatusDelegate {
                 self?.status = .connected
                 let walletName = info?.wallet?.name ?? ""
                 let walletType = info?.wallet?.metadata?.shortName?.uppercased()
-                Tracking.shared?.log(event: "ConnectWallet", data: ["walletConnectionType": walletName.uppercased(), "autoReconnect": true])
+                Tracking.shared?.log(event: "ConnectWallet", data: ["walletType": walletName.uppercased(), "autoReconnect": true])
```

Comment:
any reason for the rename? It's better, but we'll create a fork in our data consistency if we update this.

Reply:
The web code is using "walletType".   WalletConnectionType is the SDK being used to connect the wallet, whereas WalletType is the actual wallet type.